homeassistant/components/roon/media_player.py

```diff
@@ -478,6 +508,110 @@ def play_media(self, media_type, media_id, **kwargs):
                 media_id,
             )
 
+    async def async_join(self, join_ids):
+        """Add another Roon player to this player's join group."""
+
+        zone_data = self._server.roonapi.zone_by_output_id(self._output_id)
```

Comment:
Does this do I/O? If so, instead of calling into the executor twice, just make this method sync (drop `async`)

Reply:
It does - so have done as you suggested.